sway/desktop/xwayland.c

```diff
@@ -1,4 +1,5 @@
 #define _POSIX_C_SOURCE 199309L
+#ifdef HAVE_XWAYLAND
```

Comment:
Can you exclude this file if xwayland is disabled? You can do this with `get_option` in the meson file.